src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -277,8 +277,7 @@ uint64_t AccountStoreSC<MAP>::InvokeEvmInterpreter(
         try {
           if (it->hasCode() && it->Code().size() > 0) {
             targetAccount->SetImmutable(
-                DataConversion::StringToCharArray("EVM" + it->Code()),
-                contractAccount->GetInitData());
+                DataConversion::StringToCharArray("EVM" + it->Code()), {});
```

Comment:
Is it ever possible that a smart contract would want to modify the contents of code retaining the previous value of data, for instance an account could have both scilla contracts and evm contracts and potentially an evm contract could change the contents of the scilla code field which is just a string and preserver whatever value of data was currently there.

Reply:
There's just no "data" in EVM accounts.
There's only balance, nonce, and storage (root).  If/when we start mixing EVM/Scilla in the same contract, we'll have to deal with this, but then we can't just all it SetImmutable anymore.